packer/http/ks.cfg

```diff
@@ -3,7 +3,7 @@ cdrom
 lang en_US.UTF-8
 keyboard us
 unsupported_hardware
-network --bootproto=dhcp --device=eth0 --onboot=on --hostname=centos7-katello-devel-stable.example.com
+network --bootproto=dhcp --device=eth0 --onboot=on --hostname=almalinux8-katello-devel-stable.example.com
```

Comment:
Shouldn't the devel box be updated to `centos9-stream`?

Reply:
@ekohl looking at here https://github.com/theforeman/forklift/blob/71256504f735e59792e7b7179b06bd9b1c7a309b/vagrant/boxes.d/99-local.yaml.example#L100-L105 Do you want me to update it to `centos9-stream` instead of `almalinux8`. I might not able to connect something here, would you mind expanding it more on this?